aspnetcore/mvc/views/tag-helpers/built-in/index.md

```diff
@@ -3,18 +3,21 @@ title: ASP.NET Core built-in Tag Helpers
 author: pkellner
 description: Find out how ASP.NET Core built-in Tag Helpers boost your productivity.
 ms.author: riande
-ms.date: 09/18/2018
+ms.custom: mvc
+ms.date: 10/10/2018
 uid: mvc/views/tag-helpers/builtin-th/Index
 ---
 
 # ASP.NET Core built-in Tag Helpers
 
 By [Peter Kellner](http://peterkellner.net)
 
-ASP.NET Core includes many built-in Tag Helpers to boost your productivity. This section provides an overview of the built-in Tag Helpers.
+ASP.NET Core includes built-in Tag Helpers to boost your productivity. This section provides a list of the built-in Tag Helpers.
```

Comment:
@tdykstra 	please review.

Reply:
I'll just load Damien's list for now. We can adjust on the next pass.